Taxi/Taxi/Presenter/CalendarView.swift

```diff
@@ -0,0 +1,194 @@
+                            Capsule()
+                                .strokeBorder(data.date.isSameDay(selectedDate) ? Color.customYellow : todayCapsuleBorder(data.date, borderColor: Color.darkGray))
+                        }
+                            .padding(.horizontal, 6)
+                    )
+                    .onTapGesture {
+                        selectedDate = data.date
+
+                        switch calendarType {
+                        case .modalFilter:
+                            guard taxiParties.first(where: {party in
+                                data.date.isSameDay(intDateConverter.makeDateType(from: party.meetingDate))
+                            }) == nil else { return }
+                                action()
+                        case .addParty:
+                            action()
+                        }
+                    }
+                    .disabled(data.monthType == .unparticipable)
+            }
+        }
+    }
+
+    // MARK: - function
+
+    // 화살표 이동에 따른 달 이동
+    private func changeCurrentMonth() -> Date {
+        let calendar = Calendar.current
+        return calendar.date(byAdding: .month, value: currentMonth, to: Date()) ?? Date()
+    }
+
+    // 해당 달 에서의 날짜 계산
+    private func getExactDates() -> [DayContainer] {
+        let calendar = Calendar.current
+        let current = changeCurrentMonth()
+        let components = calendar.dateComponents([.year, .month], from: current)
+        let firstDayIndexOfWeek = calendar.component(.weekday, from: calendar.date(from: components) ?? Date())
+        let currentMonthDays = current.monthlyDayCount()
+
+        var thisMonthDates = current.getMonthDates().map { date in
+            DayContainer(day: date.day, date: date, monthType: checkMonthType(date))
+        }
+
+        for _ in 0..<firstDayIndexOfWeek - 1 {
+            thisMonthDates.insert(DayContainer(day: 0, date: Date(), monthType: .unparticipable), at: 0)
+        }
+        oneWeekAppend(firstDayIndexOfWeek, currentMonthDays, &thisMonthDates)
+
+        return thisMonthDates
+    }
+
+    // 다섯줄의 달력을 여섯줄로 만들기 위해 임의의 데이터를 넣어주는 작업
+    private func oneWeekAppend(_ firstDayOfWeek: Int, _ monthlyDayCount: Int, _ monthDates: inout [DayContainer]) {
+        let lastDayIndex = firstDayOfWeek - 1 + monthlyDayCount
+        if lastDayIndex < 36 {
+            for _ in 0..<(36 - lastDayIndex) {
+                monthDates.insert(DayContainer(day: 0, date: Date(), monthType: .unparticipable), at: monthDates.endIndex)
+            }
+        }
+    }
+
+    // 해당 날짜가 오늘의 날짜와 한달 차이나는지 monthType 결정
+    private func checkMonthType(_ date: Date) -> MonthType {
+        return date.isOutOfMonth() ? .unparticipable : .participable
+    }
+
+    private func todayCapsuleBorder(_ date: Date, borderColor: Color) -> Color {
+        return date.isToday() ? .gray : .clear
+    }
+
+    // MARK: - view maker
+    private func dayCell(_ value: DayContainer) -> some View {
```

Comment:
```suggestion
    private func makeDayCell(_ value: DayContainer) -> some View {
```

메소드 이름으ㅣ 시작은 동사로 하는걸 추천드립니다 ✨

Reply:
저는 개인적으로 얘는 뷰라서 동사가 아닌게 더 어울린다고 생각하는데 혹시 동사를 추천하는 이유가 있으실까요??

```swift
LazyVGrid {
    ForEach(...) { data in
        dayCell(data)
            .background(...)
    }
}
```

```swift
LazyVGrid {
    ForEach(...) { data in
        makeDayCell(data)
            .background(...)
    }
}
```